.github/workflows/ci-test-go.yml

```diff
@@ -85,6 +85,10 @@ jobs:
           #           takes precedence over all other caching options.
           skip-cache: true
 
+      - name: generate
+        working-directory: ./${{ inputs.project-directory }}
+        run: make generate
```

Comment:
I think we want to verify there are no modified files after this step

Reply:
Yep, saw you had mod tidy in there and assumed it was doing that, but on checking it's not, so added to both.